content/chainguard/libraries/java/build-configuration.md

```diff
@@ -371,6 +371,34 @@ repositories {
 }
 ```
 
+The following listing shows a valid `init.gradle` file. It wraps the
+`repositories` element with `allprojects` so that the scope of the file affects
+all projects built locally with Gradle. It also allows for `buildscript`
+download from the remote URL. Lastly the example shows use of an internal
+repository manager that only serves artifacts without authentication using HTTP
+only. Since this is not advisable unless other networking setups allow a secure
+use with HTTP, the override with the property `allowInsecureProtocol` is
+required:
+
+```
```

Comment:
One thing to be aware of is that if you don't add a label to code blocks they will be labeled with `Output` once the article is published. Not sure what you'd want to put here but I think you ought to add some label so that doesn't happen.

Reply:
Thanks for the help on slack with that ..